deployments/local/dev/docker-compose-keycloakx.yml

```diff
@@ -54,6 +54,7 @@ services:
       - "start-dev"
       - "--http-enabled=true"
       - "--http-port=8080"
+      - "--https-client-auth=request"
```

Comment:
According to https://www.keycloak.org/server/enabletls
This only validates a client certificate if the request contains one. As long as we don't set require client auth it's fine.